sections/main-product.liquid

```diff
@@ -71,12 +71,25 @@
             {%- assign media = host_product.featured_media -%}
             <li class="product__media-item grid__item slider__slide{% if media.media_type != 'image' %} product__media-item--full{% endif %}{% if section.settings.hide_variants and variant_images contains media.src %} product__media-item--variant{% endif %}" data-media-id="{{ section.id }}-{{ media.id }}">
               {% render 'product-thumbnail', media: media, position: 'featured', loop: section.settings.enable_video_looping, modal_id: section.id, xr_button: true %}
+              {%- if section.settings.show_image_label -%}
+              <div class="image_label">
+                <div id="image_type">Stock Photo</div>
+              </div>
+              {%- endif -%}
             </li>
           {%- endif -%}
           {%- for media in product.media -%}
             {%- unless media.id == product.selected_or_first_available_variant.featured_media.id -%}
               <li class="product__media-item grid__item slider__slide{% if media.media_type != 'image' %} product__media-item--full{% endif %}{% if section.settings.hide_variants and variant_images contains media.src %} product__media-item--variant{% endif %}" data-media-id="{{ section.id }}-{{ media.id }}">
                 {% render 'product-thumbnail', media: media, position: forloop.index, loop: section.settings.enable_video_looping, modal_id: section.id, xr_button: true %} 
+                {%- if section.settings.show_image_label and forloop.index == 1 -%}
+                <div class="image_label">{%- if product.metafields.kameratori.listing_facts.last.value.image_source != "database|actual" or product.metafields.kameratori.listing_facts.value.last.image_source == null or product.metafields.kameratori.listing_facts == blank -%}
```

Comment:
This is another case of my sloppy explanation I guess. When I said `image_source != "database|actual"`, I meant the | as a logical OR, so the actual string will be "database" (for images taken from database item) or "actual" (for images of the actual item), depending on which case is true. 

The line is super long and hard to read. Does liquid allow splitting this logic statement on multiple rows?

Reply:
Would not say sloppy explanation, might have being sloppy readings understanding from my side. But now it is changed. Seemed that the logic worked still okay eventhough it is on multiple rows.